ci/requirements_dev.txt

```diff
@@ -6,5 +6,5 @@ moto
 pytest>=3.1
 python-dateutil>=2.5.0
 pytz
-setuptools
-sphinx
\ No newline at end of file
+setuptools>=3.3
```

Comment:
this is fine as this is a pretty old version. can you add this to the conda recipe as well.

Reply:
can you update this?

Reply:
What are you looking to have updated?

Reply:
pandas/conda.recipe/meta.yml